.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -42,14 +42,22 @@ jobs:
     outputs:
       PR_SHA: ${{ steps.sha.outputs.result }}
 
+  test_matrix:
+    name: Generate Test Matrix
+    if: ${{ false && github.repository == 'intel/llvm' }}
+    uses: ./.github/workflows/sycl_gen_test_matrix.yml
+    with:
+      lts_config: "win_l0_gen12"
+      cts_config: "cuda"
```

Comment:
Please, remove this config. We don't have Windows machines with NVIDIA GPU.

Reply:
Done.